lib/ts/recipe/dashboard/api/userroles/permissions/removePermissions.ts

```diff
@@ -9,8 +10,16 @@ const removePermissionsFromRole = async (
     options: APIOptions,
     __: any
 ): Promise<{
-    status: "OK" | "UNKNOWN_ROLE_ERROR";
+    status: "OK" | "UNKNOWN_ROLE_ERROR" | "FEATURE_NOT_ENABLED_ERROR";
 }> => {
+    try {
+        UserRolesRecipe.getInstanceOrThrowError();
+    } catch (_) {
+        return {
+            status: "FEATURE_NOT_ENABLED_ERROR",
```

Comment:
i thought you are not going to do this in any API since it's done in the entry point API?